web/src/engine/osk/src/views/oskView.ts

```diff
@@ -625,14 +625,18 @@ export default abstract class OSKView
       return;
     }
 
+    const vkbd = this.vkbd;
     try {
       this.deferLayout = true;
-      if(this.vkbd) {
-        this.vkbd.deferLayout = true;
+      if(vkbd) {
+        vkbd.deferLayout = true;
       }
       closure();
     } finally {
       this.deferLayout = false;
+      if(vkbd) {
+        vkbd.deferLayout = false;
+      }
       if(this.vkbd) {
         this.vkbd.deferLayout = false;
       }
```

Comment:
Is it intentional to have both `if(vkbd)` and `if(this.vkbd)` here? It should have a comment if so.